### DISCUSSION TITLE
Is there non-verbose mode to sleap loading a file?

### DISCUSSION THREAD

**Discussion by lisadiez** [talmolab/sleap#1185] [Ideas]
Hello
Is there a way to turn off the metadata printing for the sleap_load command?  We have thousands of files and would like to track progress/bugs etc, and all this extra text is making it tricky to follow. Any advice?
Thanks so much!

```

slp = sleap.load_file(fname_slp)

... OUTPUTS

video_path: /vast/cm5635/cohort2/huddles/2020_07_22_15_52_33_369348_compressed_defished_shrink_cropped.mp4
video_ind: 0
provenance: {}
Saved as /mnt/b3a68699-495d-4ebb-9ab1-ac74f11c68c5/gerbil/cohort2/huddles/2020_07_22_15_52_33_369348_compressed_Day_huddle.h5
track_occupancy: (1, 28802)
point_scores: (28802, 1, 1)
instance_scores: (28802, 1)
tracking_scores: (28802, 1)
labels_path: None
```

**Comment by roomrys** [comment]
Hi @lisadiez,

This looks like the output from writing an analysis file:
https://github.com/talmolab/sleap/blob/57ee08a7490aa950f61b14f80fa8e2d372a2ca5c/sleap/info/write_tracking_h5.py#L242-L286
Are you also exporting to analysis in your code somewhere?

There is currently no way of turning off the print statements, but that would be a nice feature. I am moving this over to the Ideas category.

Thanks,
Liezl
